testing_utilities.py

```diff
@@ -385,6 +393,15 @@ def create_test_repository():
     return new_repo
 
 
+def create_new_gradebook():
+    gm = get_managers()['gm']
+    form = gm.get_gradebook_form_for_create([])
+    form.display_name = 'a gradebook'
+    form.description = 'for testing with'
+    new_gradebook = gm.create_gradebook(form)
+
+    return new_gradebook
+
```

Comment:
@xaviaracil This just needs two blank lines between the method definitions here, to meet Python standard coding style.